frontend/src/assets/sass/hwcr-animal-outcome.scss

```diff
@@ -111,6 +111,41 @@
     border-radius: 3px;
     padding: 24px;
 
+    .animal-item-edit {
```

Comment:
I remembered removing this class when doing the edit button cleanup 😆 I don't think it's using elsewhere

Reply:
I think it'll affected some of the other button icons when I tried to just get rid of it